server/data/restClient.ts

```diff
@@ -109,7 +109,7 @@ export default class RestClient {
         .set(headers)
         .end((error, response) => {
           if (error) {
-            logger.warn(sanitiseError(error), `Error calling ${this.name}, streaming path: ${path}`)
+            logger.info(`Error calling ${this.name}, streaming path: ${path}`)
```

Comment:
reducing logging when images are not being returned as this is very noisy and expected behaviour in some circumstances